### PR TITLE
update for cedar#726 (version numbers)

### PR DESCRIPTION
**Pull request by cdisselkoen**
*Issue #, if available:*

cedar-policy/cedar#726

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aaronjeline — APPROVED** [review]

**Review by aaronjeline — APPROVED** [review]

File: CedarJavaFFI/Cargo.toml [hunk 1]
```diff
@@ -4,23 +4,22 @@ name = "cedar-java-ffi"
 edition = "2021"
 
 
-version = "3.0.0"
+version = "4.0.0"
 
 [dependencies]
+cedar-policy = { version = "4.0.0", path = "../../cedar/cedar-policy" }
```
[the remark this answers is not in the document]
Comment (cdisselkoen) [reply CedarJavaFFI/Cargo.toml:10]: This is pre-existing, let's make an issue to fix this in a different PR

File: CedarJavaFFI/Cargo.toml [hunk 2]
```diff
@@ -4,23 +4,22 @@ name = "cedar-java-ffi"
...
 serde = { version = "1.0", features = ["derive", "rc"] }
 serde_json = "1.0"
-cedar-policy = { version = "3.0", path = "../../cedar/cedar-policy" } # Need latest version from github
+thiserror = "1.0.49"
+itertools = "0.11.0"
```
[the remark this answers is not in the document]
Comment (cdisselkoen) [reply CedarJavaFFI/Cargo.toml:14]: Was about to fix this but @shaobo-he-aws merged the PR before I had a chance. Feel free to make a followup

[the remark this answers is not in the document]
Comment (shaobo-he-aws) [reply CedarJavaFFI/Cargo.toml:14]: Follow up PR: https://github.com/cedar-policy/cedar-java/pull/107

File: CedarJavaFFI/Cargo.toml (hunk 1 above)
[the remark this answers is not in the document]
Comment (shaobo-he-aws) [reply CedarJavaFFI/Cargo.toml:10]: Follow up PR: https://github.com/cedar-policy/cedar-java/pull/107

**Pull request merged**